src/js/components/ThumbsRating/ThumbsRating.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Like, LikeFill, Dislike, DislikeFill } from 'grommet-icons';
+import { ThumbsRatingPropTypes } from './propTypes';
+import { RadioButtonGroup } from '../RadioButtonGroup';
+
+const ThumbsRating = ({ color, options, ...rest }) => {
+  const likeOption = options ? options[0] : 'like';
+
+  let colorProp;
+  let colorfill;
+  let coloroutline;
+  if (color && typeof color === 'string') {
+    colorProp = color;
+  } else if (color && typeof color === 'object') {
+    colorfill = color.fill;
+    coloroutline = color.outline;
+  } else colorProp = undefined;
+
+  return (
+    <RadioButtonGroup
+      direction="row"
+      options={options || ['like', 'dislike']}
```

Comment:
Given how the child objects are created, I don't think we should allow customizing the `options` via a property.